coeus-impl/pom.xml

```diff
@@ -668,5 +668,21 @@
             <groupId>com.thoughtworks.xstream</groupId>
             <artifactId>xstream</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Do you compile against any of this?  If you don't then it should be runtime scope.